demo/index.html

```diff
@@ -107,6 +107,36 @@
         </style>
       </template>
     </demo-snippet>
+
+    <h3>Badge icons can be resized</h3>
+    <demo-snippet class="centered-demo">
+      <template>
+        <!-- Enlarged paper-badge with no icon resizing. -->
+        <paper-icon-button id="icon3" icon="communication:email" alt="inbox">
+        </paper-icon-button>
+        <paper-badge icon="social:mood" for="icon3" label="happy">
+        </paper-badge>
+
+        <!-- Enlarged paper-badge with icon resizing. -->
+        <paper-icon-button id="icon4" icon="account-box" alt="account-box">
+        </paper-icon-button>
+        <paper-badge icon="social:mood" for="icon4" label="happy" class="large-icon">
+        </paper-badge>
+
+        <style is="custom-style">
+          paper-badge {
```

Comment:
i think this style is actually global, so you might want to do something like `paper-badge.large` instead